app/src/main/java/com/github/houseorganizer/houseorganizer/util/Util.java

```diff
@@ -20,4 +34,49 @@ public static boolean putEventStringsInData(Map<String, String> event, Map<Strin
         }
         return false;
     }
```

Comment:
This method might be able to be moved into the Event class also, since it is about events

Reply:
True